### PR TITLE
ARXML: make it work even if a signal's compu method does not specify a category

### PR DESCRIPTION
**Pull request by andlaus**
in this case, we simply assume the raw value of the signal to be the physical one.

this hopefully fixes [#310](https://github.com/eerimoq/cantools/issues/310).

Andreas Lauser <andreas.lauser@mbition.io>, Mercedes-Benz AG on behalf of [MBition GmbH](https://mbition.io/).

[Imprint](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md#mbition-gmbh)

### CODE REVIEW

**Comment by andlaus** [comment]
the test failures seem to be due to a change in the CI environment, i.e., unrelated to this PR:
```
for f in tests/files/c_source/motohawk.c tests/files/c_source/padding_bit_order.c tests/files/c_source/vehicle.c tests/files/c_source/multiplex.c tests/files/c_source/multiplex_2.c tests/files/c_source/floating_point.c tests/files/c_source/no_signals.c tests/files/c_source/choices.c tests/files/c_source/signed.c tests/files/c_source/my_database_name.c tests/files/c_source/min_max_only_6_0.c tests/files/c_source/abs.c ; do \
    gcc-8 -Wall -Wextra -Wdouble-promotion -Wfloat-equal -Wformat=2 -Wshadow -Werror   -Wconversion -Wpedantic -std=c99 -O3 -c $f ; \
done
/bin/sh: 2: gcc-8: not found
```

**Comment by andlaus** [comment]
#316 fixes this...

File: cantools/database/can/formats/arxml.py [hunk 1]
```diff
@@ -674,9 +674,14 @@ def _load_system_signal(self, system_signal, decimal, is_float):
...
-                    'CATEGORY in compu method {} does not exist.'.format(
-                        compu_method.find('SHORT-NAME').text))
+                # if no category is specified, we assume that the
+                # physical value of the signal corresponds to its
+                # binary representation.
+                return (minimum,
+                        maximum,
+                        factor,
+                        offset,
+                        choices)
```
Comment (H4rryK4ne) [inline cantools/database/can/formats/arxml.py:684]: ```unit``` and ```comments``` need also be returned in this tuple

**Comment by H4rryK4ne** [comment]
Here is a fixed example file: [compu_method_no_category.zip](https://github.com/eerimoq/cantools/files/6782347/compu_method_no_category.zip) to test the branch

File: cantools/database/can/formats/arxml.py (hunk 1 above)
Comment (andlaus) [reply cantools/database/can/formats/arxml.py]: oops, right. Fixed, thanks for the catch!

**Comment by andlaus** [comment]
> Here is a fixed example file: [compu_method_no_category.zip](https://github.com/eerimoq/cantools/files/6782347/compu_method_no_category.zip) to test the branch

Thanks. Do you mind if I made a test case out of it? (will probably take until next week.)

**Comment by H4rryK4ne** [comment]
> Thanks. Do you mind if I made a test case out of it? (will probably take until next week.)

Feel free.

**Comment by andlaus** [comment]
alright; I've just added the unit test for this, so I suppose this PR is now complete. (note that in order for the CI system to work #316 needs to be merged first...)

**Comment by juleq** [comment]
@andlaus Not sure why you referenced #333 here. Seems unrelated.

However, should I merge this?

**Comment by andlaus** [comment]
> Not sure why you referenced #333 here

the reference was backwards :). github automatically adds mentions to issues and PRs referenced elsewhere. In the context of #333, I mentioned this PR as an example how an issue should be reported and fixed...

> should I merge this?

Please do. (I would also be deeply obliged to you if you went over the other open PRs and merged the ones which you deem ready...)

**Comment by juleq** [comment]
I see. Makes sense :). Ok, will look at the others later.

**Pull request merged**